lua/browse/bookmarks.lua

```diff
@@ -29,6 +29,10 @@ M.search_bookmarks = function(config)
         end
     end
 
+    local function get_domain(url)
+        return string.match(url, "https?://([^/]+)")
+    end
```

Comment:
You should move this to utils and please add some `docstring` explaining the argument and return

Reply:
I put it in `utils.lua`, removed it from `bookmarks.lua` and added the documentation